### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by Ugboaja-Uchechi**
Add styling to the application to make it match this wireframe. Built with bootstrap.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,39 +5,31 @@
...
-        <input type="text" id="isbn" class="u-full-width" required>
+      <div class="w-50">
+        <input type="text" id="isbn" class="mb-3 p-1 w-100 border-dark text-dark" placeholder="ISBN" required>
       </div>
-      <div>
-        <input type="submit" value="Add" class="u-full-width">
+      <div class="d-flex align-items-center w-50 justify-content-end">
+        <input type="submit" value="Add" class="border-dark text-dark mb-3">
       </div>
     </form>
```
Comment (DeliceLydia) [inline index.html:32]: - [ ] Please add a `navigation` bar that has 3links: booklist, add book form, and contact info
- [ ] And you need to add a date below the navigation bar and you should use [Luxon](https://moment.github.io/luxon/#/) to get the current date and time
- [ ] Remember to add a `footer` as well

Comment (DeliceLydia) [reply index.html:32]: Check out the design image for reference
nsv

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Ugboaja-Uchechi) [reply index.html:32]: I accidentally submitted day3 project for day 4

**Pull request merged**